xml/depl_maintenance.xml

```diff
@@ -1503,6 +1503,295 @@ migrate</screen>
+	other VMs to make room for migration. Note that the Bash shell script
+	that starts the live migration for the &compnode; N is executed from
+	the &contrnode; M. Check the
+	<filename>/var/log/crowbar/node-upgrade.log</filename> file on the
+	&contrnode; for the information that can help you with
+	troubleshooting. You might also need to check &ostack-bare; logs in
+	<filename>/var/log/nova</filename> on the &compnode; as well as on the
+	&contrnode;s.
+       </para>
+       <para>
+	It is possible that live-migration of a certain VM takes too long. This
+	can happen if instances are very large or network connection between
+	compute hosts is slow or overloaded. If this case, try to raise the
+	global timeout in
+	<filename>/etc/crowbar/upgrade_timeouts.yml</filename>.  
+       </para>
+       <para>
+	We recommend to perform the live migration manually first. After it is
+	completed successfully, call the <command>crowbarctl upgrade</command> command.
+       </para>
+       <para>
+	The following commands can be helpful for analyzing issues with live migrations:
+       </para>
+       <screen>
+	nova server-migration-list / server-migration-show
+        nova instance-action-list / instance-action
+       </screen>
+       <para>
+	Note that these commands require &ostack-bare; administrator privileges.
+       </para>
+       <para>
+	The following log files may contain useful information:
+       </para>
+       <itemizedlist>
+	<listitem>
+	 <para>
+	  <filename>/var/log/nova/nova-compute</filename> on the source and
+	  target &compnode;s.
+	 </para>
+	</listitem>
+	<listitem>
+	 <para>
+	  <command>/var/log/nova/*.log</command> (especially log files for the
+	  conductor, scheduler and placement services) on the &contrnode;s.
+	 </para>
+	</listitem>
+       </itemizedlist>
+       <para>
+	It can happen that active instances and instances with heavy
+	loads cannot be live migrated in a reasonable time. In that case,  you
+	can abort a running live-migration using the <command>nova
+	live-migration-about <replaceable>MIGRATION-ID</replaceable></command>
+	command. You can then perform the upgrade of the specific node at a
+	later time.
+       </para>
+       <para>
+	Alternatively, it is possible to force the completion of
+	the live migration by using the <command>nova
+	live-migration-force-complete
+	<replaceable>MIGRATION-ID</replaceable></command> command. However,
+	this might pause the instances for a prolonged period of time and have
+	a negative impact on the workload running inside the instance.
+       </para>
+      </answer>
+     </qandaentry>
+     <qandaentry>
+      <question>
+       <para>
+	Node has failed during OS upgrade.
+       </para>
+      </question>
+      <answer>
+       <para>
+	Possible reasons include an incorrect repository setup or package
+	conflicts. Check the <filename>node-upgrade.log</filename> log file on the
+	node. Check the repositories on node using the <command>zypper
```

Comment:
problematic/affected node ?

Reply:
I'm not sure where in the text _problematic/affected node_ should be.

Reply:
Check the <filename>node-upgrade.log</filename> log file on the _node_

Just wanted to make sure we know what node should be checked